content/en/platform/corda/1.3/cenm/deployment-kubernetes-idman.md

```diff
@@ -34,10 +34,10 @@ helm install cenm-idman idman --set idmanPublicIP=X.X.X.X --set prefix=cenm --se
 | Parameter                     | Description                                              | Default value         |
 | ----------------------------- | -------------------------------------------------------- | --------------------- |
 | `bashDebug`                   | Display additional information while running bash scripts (useful while investigating issues) | `false` |
-| `dockerImage.name`            | URL to Identity Manager Docker image used by the Identity Manager Service Helm chart | `acrcenm.azurecr.io/identitymanager/identitymanager` |
-| `dockerImage.tag`             | Docker image Tag for the Docker image used by the Identity Manager Service Helm chart | `1.3` |
+| `dockerImage.name`            | URL to Identity Manager Docker image used by the Identity Manager Service Helm chart | `corda/enterprise-identitymanager` |
+| `dockerImage.tag`             | Docker image Tag for the Docker image used by the Identity Manager Service Helm chart | `1.3.5-zulu-openjdk8u242` |
 | `dockerImage.pullPolicy`      | Docker image pull policy for the Docker image used by the Identity Manager Service Helm chart. More info: https://kubernetes.io/docs/concepts/containers/images/#updating-images | `Always` |
-| `dockerImageCli.name`            | URL to Identity Manager Docker image used by the CENM Command-Line (CLI) tool Helm chart | `acrcenm.azurecr.io/cli/cli` |
+| `dockerImageCli.name`            | URL to Identity Manager Docker image used by the CENM Command-Line (CLI) tool Helm chart | `corda/enterprise-cenm-cli` |
 | `dockerImageCli.tag`             | Docker image Tag for the Docker image used by the CENM Command-Line (CLI) tool Helm chart | `1.3` |
```

Comment:
The tag for `corda/enterprise-cenm-cli` should be updated as well. Can we please make it `1.3.5-zulu-openjdk8u242` in all 1.3 files? Also for 1.4 and 1.5 (1.4.4 and 1.5.4 respectively)